src/System.Linq.Expressions/src/System/Linq/Expressions/IndexExpression.cs

```diff
@@ -272,7 +272,7 @@ private static string GetArgTypesString(Expression[] arguments)
 
         private static PropertyInfo FindProperty(Type type, string propertyName, Expression[] arguments, BindingFlags flags)
         {
-            var props = type.GetProperties(flags).Where(x => x.Name.Equals(propertyName, StringComparison.CurrentCultureIgnoreCase)); ;
+            var props = type.GetProperties(flags).Where(x => x.Name.Equals(propertyName, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
Unrelated, but is there a reason why this uses `StringComparison.CurrentCultureIgnoreCase` over `OrdinalIgnoreCase`?